domain/transfer/sql_repository.go

```diff
@@ -108,6 +132,8 @@ const transferSchema = `
 		status char(10) NOT NULL DEFAULT 'created',
 		description varchar(200) NOT NULL,
 	    dossier_id char(36) NOT NULL,
+	    fhir_advancenotice_composition VARCHAR(100) NOT NULL,
+	    fhir_nursinghandoff_composition VARCHAR(100) NULL,
```

Comment:
Both will contain JSON encoded data right? This should be "ok" for demo purposes I think but it would probably be better to create two more tables and add two one-to-one relationships. What do you think?

Reply:
Nope, they contain the reference to the FHIR resource that contains the messages (e.g. `Composition/some-uuid`)

Reply:
Check!